src/cache/lru_cache_manager/lru_queue.h

```diff
@@ -21,15 +22,16 @@ typedef struct LRUQueue {
   LRUNode *tail;            // Queue tail
   LRUNode *emptySpace;      // Next empty place in the queue
   bool fullCapacity;        // Indication if the queue was reached full cacpcity
+  LRUNode **emptyCells;
```

Comment:
What's the difference between emptyCells and emptySpace?

Reply:
single and multiple. renamed